test/integrations/validation/Zod.spec.ts

```diff
@@ -0,0 +1,348 @@
+
+import { expect, describe, it } from 'vitest';
+
+import zod from '../../../src/integrations/validation/module';
+
+import { SCHEMAS, ERRORS } from './_fixtures/Zod.fixtures';
+
+describe('Zod', () =>
+{
+    describe('should validate using the String schema', () =>
```

Comment:
Descriptions typically do not start with `should`. I think the description `String values` would suffice here.